qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/20_date_range.yml

```diff
@@ -42,8 +38,7 @@
 "Create index with joda style index that is incompatible with java.time (>6.1)":
   - skip:
       features: "allowed_warnings"
-      version: " - 6.8.0, 7.0.0 -"
-      reason: change of warning message, we skip 7 becase this format will be considered java
```

Comment:
Since this test should be skipped on all versions newer than 7.0.0, I think we should remove the test.